### ISSUE TITLE
Question About the PMR dataset

### ISSUE THREAD

**Issue by Gary-code** [YunxinLi/Multimodal-Context-Reasoning#7]
In the original PMR paper: Premise-based Multimodal Reasoning: Conditional Inference on Joint Textual and Visual Clues. The dataset is divided into Original and Adversarial types, which consist of 12, 080 samples respectively. In your paper, which kind of dataset do you use? Looking forward to your reply.

**Comment by YunxinLi** [comment]
we only use the original dataset. adversarial dataset. ---- Replied Message ----FromGary ***@***.***>Date09/18/2023 17:33 ***@***.***> ***@***.***>Subject[YunxinLi/Multimodal-Context-Reasoning] Question About the PMR dataset (Issue #7)
In the original PMR paper: Premise-based Multimodal Reasoning: Conditional Inference on Joint Textual and Visual Clues. The dataset is divided into Original and Adversarial types, which consist of 12, 080 samples respectively. In your paper, which kind of dataset do you use? Looking forward to your reply.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/YunxinLi/Multimodal-Context-Reasoning/issues/7",
"url": "https://github.com/YunxinLi/Multimodal-Context-Reasoning/issues/7",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by YunxinLi** [comment]
We only use the original version of PMR dataset.

**Comment by Gary-code** [comment]
After how many epochs will the accuracy converge? It seems that in the first epoch the acc is extremely low, just about 30%?

**Comment by Gary-code** [comment]
I tried to train from scratch or start training as you said in [issue](https://github.com/YunxinLi/Multimodal-Context-Reasoning/issues/6), and I found that the AUC after each epoch of training was very oscillating. The first epoch may reach 70%, and the second epoch will drop to 20%. After training for about 15 epochs overnight, the AUC of the validation set was almost maintained at 25-30%. Is there anything I didn’t notice during the training process? Looking forward to your reply.

**Comment by YunxinLi** [comment]
During training, our model achieve best performance on about 4-6 epochs.
1. you could try freezinng the parameters of VLMs, and only tune the parameters of mapping networks and language models.
2. learning rate or other super-parameters.
3. this issue is strange, yet I could make sure that acc has not such a big shock.
